packages/twenty-server/src/modules/workflow/workflow-runner/workflow-runner.service.ts

```diff
@@ -68,6 +75,8 @@ export class WorkflowRunnerService {
       });
     }
 
-    return result.error;
+    return {
+      error: result.error,
```

Comment:
this is not our throw TwentyException pattern, we should create a WorkflowRunnerException class, throw it and catch it from the caller side